cli/cli/cli.go

```diff
@@ -452,6 +457,49 @@ func (c *CLI) logLevel() string {
 	return c.config.GetString("rexray.logLevel")
 }
 
+// handles the known_hosts error,
+// if error is ErrKnownHosts, stop execution to prevent unstable state
+func (c *CLI) handleKnownHostsError(err error) error {
```

Comment:
Hi @vladimirvivien,

I know you've updated the code since I made my previous remark, but this function still pushes the 80char boundary and could be easily fixed if you'd invert the if logic. These nested ifs are not necessary. Just return early.